topo_processor/util/checksum.py

```diff
@@ -0,0 +1,14 @@
+import hashlib
+
+import multihash
+
+
+def multihash_as_hex(file_path, hash_method: str = "sha2-256") -> str:
```

Comment:
`file_path` missing type
`hash_method` is not respected as you  just `hashlib.sha256()`

`BLOCK_SIZE` should not be defined inside the function


The actual iteratorion of the file could be simplified to
```
with open('file.name', 'rb') as file:
    while chunk := file.read(CHUNK_SIZE)
        file_hash.update(chunk)
```

Reply:
wow, good to know what `:=` does

Reply:
:= is new in python 3.8 - and a great improvement it is :-)